### PR TITLE
Display readme file if available

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**
Display a readme file if it exists under the root of the current reference.
## Visual
Screen Shot 2022-07-26 at 20 09 26

### CODE REVIEW

File: webui/src/pages/repositories/repository/objects.jsx [hunk 1]
```diff
@@ -291,6 +296,31 @@ const TreeContainer = ({
     );
 }
 
+const READMEContainer = ({repo, reference}) => {
+    const {response, error, loading} = useAPI(async () => {
+        return await objects.get(repo.id, reference.id, 'README.md');
+    });
+
+    if (loading || error) {
+        return <></>
```
Comment (arielshaqed) [inline webui/src/pages/repositories/repository/objects.jsx:305]: nit:
```suggestion
        return <></>;
```

Comment (arielshaqed) [inline webui/src/pages/repositories/repository/objects.jsx:305]: Ideally we would display any non-404 errors: if we got here and there _is_ a README.md, the user might be expecting to see it!

**Review by arielshaqed — APPROVED** [review]
# NEAT!

I'm assuming remark and react-markdown are well-regarded packages.

I would be happy if we could lock their versions down a bit more: these packages take actual user data and display it to other users, which might be rather more than we've had so far in the GUI.  @Jonathan-Rosenberg + @nopcoder WDYT -- am I just being paranoid here?

File: webui/src/pages/repositories/repository/objects.jsx (hunk 1 above)
Comment (johnnyaug) [reply webui/src/pages/repositories/repository/objects.jsx:305]: null also works

File: webui/src/pages/repositories/repository/objects.jsx [hunk 2]
```diff
@@ -291,6 +296,31 @@ const TreeContainer = ({
     );
 }
 
+const READMEContainer = ({repo, reference}) => {
```
Comment (johnnyaug) [inline webui/src/pages/repositories/repository/objects.jsx:299]:
```suggestion
const ReadmeContainer = ({repo, reference}) => {
```

**Review by johnnyaug — APPROVED** [review]
Awesome!

**Comment by johnnyaug** [comment]
Solves: #3410

**Pull request merged**